src/utils.h

```diff
@@ -44,6 +44,7 @@
 #pragma once
 
 #include <boost/uuid/detail/sha1.hpp>
+#include <ledger.hh>
```

Comment:
`ledger.hh`. contains the definition of NO_ASSERT, without its inclusion the build fails at the linker stage with missing symbols ot `ledger::debug_assert`